docs/resources/devops_scripts/export-config.sh

```
@@ -40,10 +84,11 @@ job_status=$(kubectl get job $job_name -n $ns -o jsonpath='{.status.succeeded}'
 if [[ "$job_status" == "1" ]]; then
   pod_name=$(kubectl get pods -n $ns --selector=job-name=$job_name -o jsonpath='{.items[0].metadata.name}')
   echo "Job succeeded. Tailing logs:"
-  kubectl logs -f -n "$ns" "$pod_name"
+  kubectl logs -n "$ns" "$pod_name"
   kubectl delete job "$job_name" -n "$ns" --wait=false
   exit 0
 else
   echo "Job did not complete successfully (timeout or unknown error)."
   kubectl logs -n "$ns" -l job-name=$job_name || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)